<!-- -->
<launch>

    <!-- point_cloud_receiver -->
    <node pkg="adeye" type="point_cloud_receiver.py" name="point_cloud_receiver"/>


    <!-- GPS driver -->
    <node pkg="nmea_navsat_driver" type="nmea_serial_driver" name="navsat_driver" respawn="true">
	  <param name="port" value="/dev/ttyACM0"/>
	  <param name="baud" value="9600"/>
    </node>

    <!-- gnss_broadcaster -->
    <include file="$(find adeye)/launch/gnss_broadcaster.launch"/>

    <!-- base_link to GPS -->
    <node pkg="tf" type="static_transform_publisher" name="base_link_to_gps"
        args="0.0 0.0 0.0 0.0 0.0 0.0 1.0 base_link gps 10"/>

</launch>
